okta/resource_okta_app_group_assignment_test.go

```diff
@@ -136,7 +136,7 @@ func ensureAppGroupAssignmentExists(name string) resource.TestCheckFunc {
 	}
 }
 
-func ensureAppGroupAssignmentRetained(appName string, groupName string) resource.TestCheckFunc {
+func ensureAppGroupAssignmentRetained(appName, groupName string) resource.TestCheckFunc {
```

Comment:
Can you tell me why we no longer have this defined as a `string`?

Reply:
This is basically a linter check that will say 'plz convert `appName string, groupName string` to `appName, groupName string`.